core/trino-main/src/test/java/io/trino/spiller/TestFileSingleStreamSpillerFactory.java

```diff
@@ -143,24 +143,27 @@ private Page buildPage()
         return new Page(col1.build());
     }
 
-    @Test(expectedExceptions = RuntimeException.class, expectedExceptionsMessageRegExp = "No free or healthy space available for spill")
+    @Test
     public void throwsIfNoDiskSpace()
     {
-        List<Type> types = ImmutableList.of(BIGINT);
-        List<Path> spillPaths = ImmutableList.of(spillPath1.toPath(), spillPath2.toPath());
-        FileSingleStreamSpillerFactory spillerFactory = spillerFactoryFactory(spillPaths, 0.0);
-
-        spillerFactory.create(types, bytes -> {}, newSimpleAggregatedMemoryContext().newLocalMemoryContext("test"));
+        assertThatThrownBy(() -> {
+            List<Type> types = ImmutableList.of(BIGINT);
+            List<Path> spillPaths = ImmutableList.of(spillPath1.toPath(), spillPath2.toPath());
+            FileSingleStreamSpillerFactory spillerFactory = spillerFactoryFactory(spillPaths, 0.0);
+            spillerFactory.create(types, bytes -> {}, newSimpleAggregatedMemoryContext().newLocalMemoryContext("test"));
+        }).isInstanceOf(RuntimeException.class).hasMessage("No free or healthy space available for spill");
```

Comment:
I would change the style like this. 
```java
        List<Type> types = ImmutableList.of(BIGINT);
        List<Path> spillPaths = ImmutableList.of(spillPath1.toPath(), spillPath2.toPath());
        FileSingleStreamSpillerFactory spillerFactory = spillerFactoryFactory(spillPaths, 0.0);

        assertThatThrownBy(() -> spillerFactory.create(types, bytes -> {}, newSimpleAggregatedMemoryContext().newLocalMemoryContext("test")))
                .isInstanceOf(RuntimeException.class)
                .hasMessage("No free or healthy space available for spill");
```

Subsequent `throwIfNoSpillPaths` is also.

Reply:
This makes a lot more sense, thanks!